content/100-getting-started/index.mdx

```diff
@@ -38,6 +39,8 @@ Explore our products that make it easy to build and scale data-driven applicatio
 [**<SignalStream />Prisma Pulse**](/pulse) allows you to react to database changes with type-safe model streams in real-time.
 
 
+[**<Inspect />Prisma Optimize**](/optimize/) helps you to analyze queries, generate insights and provides recommendations that can be used to make your database queries faster.
```

Comment:
[*Inspect Prisma Optimize*](/optimize/) helps you analyze queries, generate insights, and provides recommendations to make your database queries faster.